### ISSUE TITLE
Reconfigure Location input page

### ISSUE THREAD

**Issue by SJ-bond** [younginnovations/aidstream-old#48]
The Location page is too complicated (as Rolf mentioned earlier) for users who don't have a GIS background. Waiting for 2.01 before this page is upgraded will be 6 months plus. Are there some tweaks that can be be made before this that can help? Updating to 1.04 would help in the short term as several of the fields (percentage/gazateer etc) have been deprecated. As the trend is increasing towards using sub-national locations in visualisations of IATI data, it's really important we support users to give accurate information. Two ideas:

1/ Where a user uses the pin dropped on the map, it also fills in detail about location type, name and description. (The first fields on that page). This means that the user struggles to know what to fill in on the page initially. If you changed the order so that the pin map comes below the percentage box (which has been deprecated in 1.04), it would autofill the other boxes, avoiding user confusion. A note could be put against the other fields that they are not required if you used the pin map.

2/  Gazateer - most users don't know the difference between the gazateers listed (or even know what a gazateer is!). Would it be possible to include links to the gazateers in the Help menu? The links are listed here: http://iatistandard.org/codelists/GeographicVocabulary/ 
Thanks! Sarah

**Comment by rolfkleef** [comment]
Agree, this came up in our last workshop again too: people love to add locations, but don't manage to with the current interface. It's unclear which fields are required, which are filled in automatically.
